mkt/developers/views.py

```diff
@@ -1173,3 +1176,30 @@ def verify_data(self, data):
             return False
 
         return True
+
+
+class ContentRatingsPingbackV2(CORSMixin, UpdateModelMixin, GenericAPIView):
+    """Pingback API for IARC v2.
+
+    Should conform to the PushCert API spec. Assumes that the RatingList is
+    always sent with PushCert, so we don't need to call SearchCert afterwards
```

Comment:
I can do that with the CertID that is returned (and not yet stored, I still need to do this). The uuid becomes pointless once we have the CertID